pyfolio/utils.py

```diff
@@ -117,8 +118,9 @@ def get_symbol_rets(symbol):
             if datetime.now() - pd.to_datetime(
                     getmtime(filepath), unit='s') < pd.Timedelta(days=1):
                 rets = pd.read_hdf(filepath, 'df')
-        except:
-            pass
+        except Exception as e:
```

Comment:
OK. Actually, why is it of concern for us to automatically download from Yahoo if we already do that sometimes anyway? Can we not just throw the (correct) exception, and download the data? I don't think there is a reason to be concerned as long as we aren't doing this behind users' backs. Maybe we can have some message notifying users when downloads occur. Then, if they are concerned with using outside data, they can pipe in their own benchmark or F-F factors. I imagine any firm/party wanting to use pyfolio, and concerned about using external data, would first familiarize themselves with it to the extent to where they would see messages about the downloads if we show them. Thoughts?